add-collaborators.rb

```diff
@@ -0,0 +1,65 @@
+# add-collaborators.rb
+# add as collaborators anyone who comments on a given issue in a given repo
+# assumes you created and stored the appropriate OAuth token as an ENV variable called GITHUBTEACHER_TOKEN
+
+require 'octokit'
+require 'optparse'
+
+
+# Make sure arguments are specified
+ARGV << '-h' if ARGV.empty?
+
+
+# Create options hash
+options = {}
+
+
+# Parse options
+OptionParser.new do |opts|
+  opts.banner = "Usage: add-collaborators.rb [options]"
+
+  opts.on("-r", "--repo REPOSITORY", "Repository Path -- ex: githubteacher/example-repo") do |r|
+    options[:repo] = r
+  end
+
+  opts.on("-i", "--issue ISSUE", Numeric, "Issue Number -- ex: 4") do |i|
+    options[:issue] = i
+  end
+
+  opts.on_tail("-h", "--help", "Prints this help message") do |h|
+    puts opts
+    exit
+  end
+
+end.parse!
+
+puts options
+
+
+# Assign variables
+token = 'GITHUBTEACHER_TOKEN'
```

Comment:
I finally figured out where I ran into problems with this script! Moving at a quick skim before class, I saw this line and totally missed that line 47 doesn't actually use it (instead preferring an ENV token var). 

So, we should :hocho: this line, as it's a) not used and b) potentially misleading.